starter/js/atm.js

```diff
@@ -1,38 +1,99 @@
 //Begin with the document ready function
+      $(document).ready(function() {
+        document.body.style.backgroundColor = "red";
+  //On click of the depositChecking button
+      $("#depositChecking").click(function() {
+        depositChecking();
+      });
+  //On click of the withdrawChecking button
+      $("#withdrawChecking").click(function() {
+        withdrawChecking();
+      });
+  //On click of the depositSavings button
+      $("#depositSavings").click(function() {
+        depositSavings();
+      });
+  //On click of the withdrawlSavings button
+      $("#withdrawSavings").click(function() {
+        withdrawSavings();
+      });
+});
+
+  // var checkingBalance = 0;
+
+//Checking account deposit function
+
+  function depositChecking() {
+    //Get value from the amountChecking input field
+    var amount = $("#amountChecking").val();
+
+    checkingBalance += amount;
```

Comment:
What is `checkingBalance` here? A variable? If so, it needs to be set. Is it supposed to be a DOM element? Then use jquery to select it from the DOM.